### DISCUSSION TITLE
Pronunciation problems and other problems with tortoise-tts

### DISCUSSION THREAD

**Discussion by kbuwel** [neonbjb/tortoise-tts#641] [General]
hi? 
I am Korean. 
I had some difficulty installing tortoise-tts. 
My system is windows 10 64bit 5900x rtx3060. 
The memory uses 64GB xmp 2.0. 
There was some difficulty because DeepSpeed was not installed. 
I heard that installation is difficult on Windows. 
So it was a bit difficult. 
I hope DeepSpeed will be removed in the next version, but that's just my wish. 
I created a folder called abc in the tortoise/voices folder and placed files longer than 10 seconds 
in it. 
python tortoise/read.py --textfile voice.txt --voice abc 
I created a voice.txt file and wrote it in Korean. 
And I ran it. 
The results have been created. 
However, the pronunciation was very different from Korean. 
Korean text played with English pronunciation. 
So I entered the following command: 
python turtle/get_conditioning_latents.py --voice abc 
Error with key: 'abc' 
I got this error message: 
The result was generated, but the pronunciation was strange, so I entered the above command and an 
error occurred. 
What should I do to solve it? 
The audio file has no background noise. 
We used data from aihub.or.kr.

**Comment by manmay-nakhashi** [comment]
@kbuwel thi model is only trained on en-us and en-UK so no support for other accents.
